src/basic/selinux-util.c

```diff
@@ -110,7 +110,7 @@ int mac_selinux_init(void) {
 
         label_hnd = selabel_open(SELABEL_CTX_FILE, NULL, 0);
         if (!label_hnd) {
-                log_enforcing_errno(errno, "Failed to initialize SELinux context: %m");
+                log_enforcing_errno(errno, "Failed to initialize SELinux labeling handle: %m");
```

Comment:
isn't the right spelling `labelling`? @keszybz?

Reply:
labeled American, labelled British. I think we try to use American in user messages, so that'd be OK.